packages/react-native-executorch/common/rnexecutorch/models/speech_to_text/SpeechToText.cpp

```diff
@@ -0,0 +1,76 @@
+#include "ReactCommon/CallInvoker.h"
+#include "rnexecutorch/models/EncoderDecoderBase.h"
+#include <memory>
+#include <rnexecutorch/models/speech_to_text/MoonshineStrategy.h>
+#include <rnexecutorch/models/speech_to_text/SpeechToText.h>
+#include <rnexecutorch/models/speech_to_text/WhisperStrategy.h>
+#include <stdexcept>
+#include <string>
+#include <vector>
```

Comment:
Again vector is included in header of `SpeachToText.h`. Make sure that you do not repeat such imports